src/app/components/Balances/index.tsx

```diff
@@ -0,0 +1,217 @@
+        stats: calculateBalanaceStats(channels, utxos)
+      });
+    }
+  }
+
+
+  render() {
+    const { fetchChannelsError, fetchUtxosError, chain } = this.props;
+    const { stats, denomination } = this.state;
+    const denomText = denominationSymbols[chain][denomination];
+
+    let content;
+    if (stats) {
+      content = (
+        <>
+          <div className="Balances-chart">
+            <Select
+              className="Balances-chart-denoms"
+              size="small"
+              value={denomination}
+              dropdownMatchSelectWidth={false}
+              onChange={this.handleChangeDenomination}
+            >
+              {typedKeys(Denomination).map(d => (
+                <Select.Option key={d} value={d}>
+                  {denominationSymbols[chain][d]}
+                </Select.Option>
+              ))}
+            </Select>        
+            <div className="Balances-chart-metrics">
+              <Statistic
+                title="Total Wallet Balance" 
+                value={parseFloat(fromBaseToUnit(stats.total, denomination))} 
+                suffix={<small>{denomText}</small>} 
+              />
+              <Statistic 
+                title={`Spendable Balance (${stats.spendablePercent}%)`} 
+                value={parseFloat(fromBaseToUnit(stats.spendable, denomination))} 
+                suffix={<small>{denomText}</small>} 
+              />
+            </div>    
+            <Tooltip
+              overlayClassName="Balances-chart-tip"
+              title={`
+                ${stats.pendingPercent}% Pending /
+                ${stats.channelPercent}% Lightning /
+                ${stats.onchainPercent}% ${blockchainDisplayName[chain]}
+              `}
+              placement="topRight"
+              arrowPointAtCenter
+            >
+              <Progress
+                className="Balances-chart-progress"
+                percent={stats.channelPercent + stats.pendingPercent}
+                type="circle"
+                strokeColor="#7642ff"
+                successPercent={Math.max(0.1, 100 - stats.channelPercent - stats.onchainPercent)}
+                trailColor="#ff9500"
+                format={() => `${stats.spendablePercent}%`}
+              />
+            </Tooltip>
+          </div>
+          <div className="Balances-stats">
+            <Statistic 
+              title="Pending" 
+              value={parseFloat(fromBaseToUnit(stats.pendingTotal, denomination))} 
+              valueStyle={{ color: '#858585' }} 
+            />
+            <Statistic 
+              title="Lightning" 
+              value={parseFloat(fromBaseToUnit(stats.channelTotal, denomination))} 
+              valueStyle={{ color: '#7642ff' }} 
+            />
+            <Statistic 
+              title={blockchainDisplayName[chain]}
+              value={parseFloat(fromBaseToUnit(stats.onchainTotal, denomination))} 
+              valueStyle={{ color: '#ff9500' }} 
+            />
```

Comment:
Would these guys with suffixes not fit? Looks a little awkward having numbers without indicators of what they are.

Reply:
I added the suffixes but needed to do a bit of trickery with the styles in order to make the content fit depending on how big the numbers are.